ruby_review_week_four.rb

```diff
@@ -1,37 +1,125 @@
 # You may NOT use Google. These questions are just to test where you are at. Get through as many of these as you can. If you don’t get through all of them it’s ok - they are designed to take longer than the available time. If you can’t remember a bit of code, write the pseudo-code that represents what you would write your Ruby code.
 
 # 1. What is a variable?
+# variable is a value that is stored in a variable name for use later in code. 
 
 # 2. What Ruby method can you use to make a loop from an array?
+# .each or .map
 
 # 3. In Ruby, what are booleans?
+# Booleans are something that equates to true or false 
 
 # 4. How do classes relate to objects?
+# Classes are the templates for when an object is created. 
 
 # 5. What is an instance variable?
+# an instance variable or an attribute is a variable relating to an object.
 
 # 6. Choose an everyday item (cup, computer, person) and give it two instance variables (which will be attributes), and one method (to describe a function of the object). Code the initialize method in full (for these attributes only), and the definition line of the instance method (but do not code the entire method).
+# class Computer
+#   def initialize(brand, model)
+#     @brand = brand
+#     @model = model
+#   end
+
+#   def describe_function
+
+#   end
+# end
 
 # 7. What is handy about using gems in Ruby?
+# They add more functionality to ruby, allows you to use more functions.
 
 # 8. Define a class called fish, and leave it empty.
+# class Fish
+
+# end
 
 # 9. How would you use that class to make three fish objects?
+# fish_one = Fish.new
+# fish_two = Fish.new
+# fish_three = Fish.new
 
 # 10. Add two attributes to your fish class, name and species, and make them accessible for reading and writing.
+# class Fish
+#   attr_accessor(name)
+#   attr_accessor(species)
```

Comment:
this is good, next time consider making your accessor's all on one line, without the brackets and with symbols

```rb
attr_accessor :name, :species
```